common/formatting/align.cc

```diff
@@ -471,8 +471,8 @@ static FormatTokenRange EpilogRange(const TokenPartitionTree& partition,
 
 // Mark format tokens as must-append to remove future decision-making.
 static void CommitAlignmentDecisionToRow(
-    TokenPartitionTree& partition, const AlignmentRow& row,
-    MutableFormatTokenRange::iterator ftoken_base) {
+    const AlignmentRow& row, MutableFormatTokenRange::iterator ftoken_base,
+    TokenPartitionTree& partition) {  // NOLINT
```

Comment:
ok to make this `*` to follow mutability convention.

Reply:
Done.